### DISCUSSION TITLE
What is Restored in Monailabel and how it is being called from json files in Monaibundle.

### DISCUSSION THREAD

**Discussion by Bala93** [Project-MONAI/MONAILabel#1317] [Q&A]
I am currently trying to run the Brats segmentation in MONAI bundle, and I am trying to visualize the segmentation map on the Slicer by overlaying it on the image. However, because of the 4d dimension nature of BraTS image, and 3d nature of the BraTS mask, i am unable to visualize. Hence, I wanted to repeat the result for 4D and view it in the BraTS. But, the operation Restored is not allowing this operation. May I know what is the use of Restore and how to control it ?

**Comment by SachidanandAlle** [comment]
you can use inverse transform from monai.. you can refer some other bundles at: https://github.com/Project-MONAI/model-zoo/tree/dev/models

**Reply by Bala93** [reply]
Thanks for your response. Will give it a try.
